include/utils/mixed_polyline_stitcher.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2024 Ultimaker B.V.
+// CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#ifndef UTILS_MIXED_POLYLINE_STITCHER_H
+#define UTILS_MIXED_POLYLINE_STITCHER_H
+
+#include "PolylineStitcher.h"
+#include "geometry/closed_lines_set.h"
+#include "geometry/open_lines_set.h"
+
+namespace cura
+{
+
+class MixedLinesSet;
+
+class MixedPolylineStitcher : public PolylineStitcher<OpenLinesSet, ClosedLinesSet, OpenPolyline, Point2LL>
```

Comment:
I'm not sure why this one is off in its own file, while the others' share one file?

Reply:
Might be acceptable though, since this is an actual _class_.

Reply:
Because again, I like header files with a single type :innocent: